partner_fax/views/res_partner.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_partner_simple_form_fax" model="ir.ui.view">
+        <field name="name">Add fax on partner</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_simple_form"/>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='function']" position="after">
+                    <field name="fax" placeholder="Fax..."/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+
+    <record id="view_partner_short_form_fax" model="ir.ui.view">
+        <field name="name">Add fax on partner</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_short_form"/>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//field[@name='function']" position="after">
```

Comment:
same here and Remove [data](https://github.com/OCA/partner-contact/pull/637/files#diff-3bcdbda29f030aded403bbdf43e36986R22)